### ISSUE TITLE
Bot crash after startup

### ISSUE THREAD

**Issue by Thurion666** [Meowhal/osu-ahr#51]
The bot crashes after startup instantly, no logs recorded or anything, tried to boot from new download with clean configs aswell, it just refuses to boot.
![image](https://user-images.githubusercontent.com/38357989/152532422-6eddc466-450e-47be-ab04-13ff0df20246.png)
![image](https://user-images.githubusercontent.com/38357989/152532505-79d088d7-3216-4086-bb17-c9f82f991fda.png)

**Comment by Meowhal** [comment]
did you run `npm install` command?
https://github.com/Meowhal/osu-ahr#setup

**Comment by Thurion666** [comment]
I did, the thing is i had the bot running before, wich made me realize i recently changed my irc password, that was the issue here, i didnt know first instance because there's no log or info in the output of the cmd while the password is wrong.

Edit before post: As i was trying as i write this, nope, it still doesn't work. 
![image](https://user-images.githubusercontent.com/38357989/152624916-5ca1e9cc-b84b-41a4-bf2d-0cfa29fa8e82.png)
Edit 2 before post: Tried new files with new config, followed the steps exactly as the CMD was telling me, and now it works. (i guess incompatibilities with old config files).
It's all good now, thank you very much.

Oh wowie that's a lot of text.
